### PR TITLE
All: rewrite helps

### PR DESCRIPTION
**Pull request by l-vincent-l**

### CODE REVIEW

File: source/ed/fare2ed.cpp [hunk 1]
```diff
@@ -56,17 +56,18 @@ int main(int argc, char * argv[])
...
     desc.add_options()
-        ("help,h", "Affiche l'aide")
-        ("version,v", "Affiche la version")
-        ("config-file", po::value<std::string>(), "chemin vers le fichier de configuration")
-        ("connection-string", po::value<std::string>(&connection_string)->required(), "parametres de connexion à la base de données: host=localhost user=navitia dbname=navitia password=navitia")
-        ("fare,f", po::value<std::string>(&fare_dir)->required(), "Repertoire des fichiers fare");
+        ("help,h", "Show this message")
+        ("version,v", "Show version")
+        ("config-file", po::value<std::string>(), "Path to configuration file")
+        ("connection-string", po::value<std::string>(&connection_string)->required(),
```
Comment (antoine-de) [inline source/ed/fare2ed.cpp:62]: a shortcut, a shortcut!

Comment (l-vincent-l) [inline source/ed/fare2ed.cpp:62]: It should be done in another PR.

**Comment by l-vincent-l** [comment]
Jenkins, test this please
